packages/js/product-editor/src/components/attribute-term-input-field/attribute-term-input-field.tsx

```diff
@@ -141,15 +138,15 @@ export const AttributeTermInputField: React.FC<
 		} );
 		setIsCreatingTerm( true );
 		try {
-			const newAttribute: ProductAttributeTerm =
-				await createProductAttributeTerm( {
-					...attribute,
-					attribute_id: attributeId,
-				} );
+			const newAttribute = await createProductAttributeTerm( {
+				...attribute,
+				attribute_id: attributeId,
+			} );
 			recordEvent( 'product_attribute_term_add_success', {
 				source: TRACKS_SOURCE,
 			} );
 			onChange( [ ...value, newAttribute ] );
+			// @ts-expect-error TODO: Investigate - is this correct?
 			invalidateResolutionForStoreSelector( 'getProductAttributes' );
```

Comment:
You are right, this is incorrect!
Should we update it to use the correct store instead?
```suggestion
			invalidateResolutionForProductAttributesStore( 'getProductAttributes' );
```
Using:
```
const {
		invalidateResolutionForStoreSelector: invalidateResolutionForProductAttributesStore,
	} = useDispatch( experimentalProductAttributesStore );
```

Reply:
Removing the line will also be fine I think given it currently doesn't do anything and doesn't seem to affect the user experience.

Reply:
I removed it with 1a242d57b2da01527e56addcb9ce0c7fa1485605